WordPress/Helpers/EscapingFunctionsTrait.php

```diff
@@ -218,9 +218,15 @@ final public function is_escaping_function( $functionName ) {
 		if ( array() === $this->allEscapingFunctions
 			|| $this->customEscapingFunctions !== $this->addedCustomEscapingFunctions['escape']
 		) {
+			/*
+			 * Lowercase all names, both custom as well as "known", as PHP treats namespaced names case-insensitively.
+			 */
+			$custom_escaping_functions = array_map( 'strtolower', $this->customEscapingFunctions );
+			$escaping_functions        = array_change_key_case( $this->escapingFunctions, \CASE_LOWER );
```

Comment:
Why are you doing the lowercasing _before_ the merge ?

Would it make sense to add a new parameter to the `RulesetPropertyHelper::merge_custom_array()` method (internal) to do the lowercasing in that method ?

Reply:
I mean, this is a public helper, so would that be considered a major breaking change?

Reply:
1. A new optional parameter is not a breaking change.
2. I'd keep the default to `false` as this ruleset helper is not just used for case _insensitive_ properties, so let's not break things which shouldn't be broken.